src/scene/container/Container.ts

```diff
@@ -863,14 +862,15 @@ export class Container extends EventEmitter<ContainerEvents & AnyEvent>
      * The coordinate of the object relative to the local coordinates of the parent.
      * @since 4.0.0
      */
-    get position(): PointLike
+    get position(): ObservablePoint
     {
         return this._position;
     }
 
-    set position(value: PointData)
+    set position(value: PointData | number)
     {
-        this._position.copyFrom(value);
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+        typeof value === 'number' ? this._position.set(value) : this._position.copyFrom(value);
```

Comment:
A number to set position seems a little strange IMO. Scale makes a lot of sense though.

Reply:
I've removed the number setter for position

skew currently doesn't have a setter, which is an entirely different issue which i need to fix 😆 

To me setting a number for pivot does make sense since its sort of like anchor, but happy to be overruled on this one

Reply:
Sure. Also what about anchor too?